spec/community.mediawiki

```diff
@@ -94,6 +94,8 @@ be sent in the [[comm.mediawiki/#Session_Authentication|connect response]].
 |-
 | duration   || int    || the duration in milliseconds of the penalization
 |-
+| accountid  || string || account ID
```

Comment:
I'm good with `accountid` being added, but we do need it in a separate PR for tracking purposes.

Reply:
eh, sorry if I was not clear enough, I was referring to hte spec/ file change.  I had no issue with the implementation here.  It's changes to the actual spec files that go in separate PRs

Reply:
I'll do it in a separate pr...